x/evmutil/keeper/evm.go

```diff
@@ -122,11 +123,15 @@ func (k Keeper) CallEVMWithData(
 
 	res, err := k.evmKeeper.ApplyMessage(ethGasContext, msg, evmtypes.NewNoOpTracer(), true)
 	if err != nil {
+		if len(res.Ret) > 0 {
```

Comment:
for troubleshooting scenerios might be useful to debug log the params passed to `ApplyMessage` in this error branch

Reply:
i'll add the log to the msg_server since it will cover other errors from evm as well and avoids polluting logs from the evm calls from other places.